### PR TITLE
Fix x-axis graph bug

### PR DESCRIPTION
**Pull request by pirlgon**
Before this commit, the x-axis would continue to update even if the data was not refreshed.
For example, this caused the graph to be off the axis after tab navigation.
   
We introduced rawFrom and rawTo in the store service and filled the from and to value with the parsed result of it.
from and to are no longer parsed out of the service.
    
In the datepicker, watch has been replaced by watchEffect for simplicity.

### CODE REVIEW

File: powa/static/js/App.vue [hunk 1]
```diff
@@ -368,8 +368,7 @@ function loadData() {
 watch(
   () => route.params,
   function (newVal, oldVal) {
-    from.value = route.query.from || defaultFrom;
-    to.value = route.query.to || defaultTo;
+    setFromTo(route.query.from || defaultFrom, route.query.to || defaultTo);
```
Comment (pgiraud) [inline powa/static/js/App.vue]: Maybe we could allow to pass an undefined value, and use defaultFrom only in the service. And then `defaultFrom` wouldn't have to be exposed.

File: powa/static/js/components/DateRangePicker/DateRangePicker.vue [hunk 2]
```diff
@@ -202,20 +202,15 @@ function loadRangeShortcut(shortcut) {
...
-);
+watchEffect(() => {
+  inputFrom.value = rawFrom.value;
+  synchronizeFromPicker();
+});
+
+watchEffect(() => {
+  inputTo.value = rawTo.value;
+  synchronizeToPicker();
+});
```
Comment (pgiraud) [inline powa/static/js/components/DateRangePicker/DateRangePicker.vue:213]: Maybe we could use only one `watchEffect`. To be tested.

File: powa/static/js/components/DateRangePicker/DateRangePicker.vue [hunk 3]
```diff
@@ -244,16 +239,16 @@ function applyPickerToDate() {
...
 }
 
 function zoomOut() {
-  const from_ = dateMath.parse(from.value);
-  const to_ = dateMath.parse(to.value);
+  const from_ = dateMath.parse(rawFrom.value);
+  const to_ = dateMath.parse(rawTo.value);
   const diff = to_ - from_;
   const newFrom = toISO(new Date(from_.toDate().getTime() - diff / 2));
   const newTo = toISO(new Date(to_.toDate().getTime() + diff / 2));
```
Comment (pgiraud) [inline powa/static/js/components/DateRangePicker/DateRangePicker.vue]: I think we could probably avoid parsing the raw values here.

File: powa/static/js/components/DateRangePicker/DateRangePicker.vue (hunk 2 above)
Comment (pirlgon) [reply powa/static/js/components/DateRangePicker/DateRangePicker.vue:213]: We will synchronise from and to, even if one is changed. Does it matter?

Comment (pgiraud) [reply powa/static/js/components/DateRangePicker/DateRangePicker.vue:213]: Sorry my bad. I forgot that we have 2 different functions for synchronization.

**Pull request merged**